pkg/operator/operator.go

```diff
@@ -187,216 +206,157 @@ func (o *operator) syncHTTPEndpoint(ctx context.Context) func(obj interface{}) {
+			_, rErr := o.secProvider.Handler(ctx)
 			if rErr != nil {
-				return fmt.Errorf("unable to create webhook Subscriptions v1alpha1: %w", rErr)
+				return rErr
 			}
-			rErr = ctrl.NewWebhookManagedBy(o.mgr).
-				For(&subscriptionsapiV2alpha1.Subscription{}).
-				Complete()
+			return o.mgr.Start(ctx)
+		},
+		func(ctx context.Context) error {
+			// start healthz server
+			if rErr := healthzServer.Run(ctx, healthzPort); rErr != nil {
+				return fmt.Errorf("failed to start healthz server: %w", rErr)
+			}
+			return nil
+		},
+		func(ctx context.Context) error {
+			if rErr := o.apiServer.Ready(ctx); rErr != nil {
+				return fmt.Errorf("API server did not become ready in time: %w", rErr)
+			}
+			healthzServer.Ready()
+			log.Infof("Dapr Operator started")
+			<-ctx.Done()
+			return nil
+		},
+		func(ctx context.Context) error {
+			sec, rErr := o.secProvider.Handler(ctx)
 			if rErr != nil {
-				return fmt.Errorf("unable to create webhook Subscriptions v2alpha1: %w", rErr)
+				return rErr
+			}
+			sec.WatchTrustAnchors(ctx, caBundleCh)
+			return nil
+		},
+		func(ctx context.Context) error {
+			sec, rErr := o.secProvider.Handler(ctx)
+			if rErr != nil {
+				return rErr
 			}
-		}
-
-		// load certs from disk
-		certChain, rErr := o.loadCertChain(ctx)
-		if rErr != nil {
-			return fmt.Errorf("failed to load cert chain: %w", rErr)
-		}
-
-		rErr = o.patchCRDs(ctx, o.mgr.GetConfig(), "subscriptions.dapr.io")
-		if rErr != nil {
-			return rErr
-		}
-
-		log.Info("Starting api server")
-		rErr = o.apiServer.Run(ctx, certChain)
-		if rErr != nil {
-			return fmt.Errorf("failed to start API server: %w", rErr)
-		}
-		return nil
-	}})
-	if err != nil {
-		return err
-	}
 
-	err = o.mgr.Add(nonLeaderRunnable{func(ctx context.Context) error {
-		if !o.mgr.GetCache().WaitForCacheSync(ctx) {
-			return errors.New("failed to wait for cache sync")
-		}
+			caBundle, rErr := sec.CurrentTrustAnchors()
+			if rErr != nil {
+				return rErr
+			}
 
-		componentInformer, rErr := o.mgr.GetCache().GetInformer(ctx, &componentsapi.Component{})
-		if rErr != nil {
-			return fmt.Errorf("unable to get setup components informer: %w", rErr)
-		}
+			for {
+				rErr = o.patchCRDs(ctx, caBundle, o.mgr.GetConfig(), "subscriptions.dapr.io")
+				if rErr != nil {
+					return rErr
+				}
+
+				select {
+				case caBundle = <-caBundleCh:
+				case <-ctx.Done():
+					return nil
+				}
+			}
+		},
+		func(ctx context.Context) error {
+			sec, rErr := o.secProvider.Handler(ctx)
+			if rErr != nil {
+				return rErr
+			}
 
-		_, rErr = componentInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
-			AddFunc: o.syncComponent(ctx),
-			UpdateFunc: func(_, newObj interface{}) {
-				o.syncComponent(ctx)(newObj)
-			},
-		})
-		if rErr != nil {
-			return fmt.Errorf("unable to add components informer event handler: %w", rErr)
-		}
-		<-ctx.Done()
-		return nil
-	}})
-	if err != nil {
-		return err
-	}
+			log.Info("Starting api server")
```

Comment:
Nit :)

"api" = "bees" in italian

```suggestion
			log.Info("Starting API server")
```